src/test/java/tictactoe/PlayerFactoryTest.java

```diff
@@ -0,0 +1,38 @@
+package tictactoe;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class PlayerFactoryTest {
+    @Test
+    public void itCreateTwoCommandLinePlayers() {
+        PlayerFactory playerFactory = new PlayerFactory(null, null);
+        Player[] players = playerFactory.getPlayers("HUMANvsHUMAN");
+        assertTrue(players[0] instanceof CommandLinePlayer);
```

Comment:
Can I suggest the use of http://joel-costigliola.github.io/assertj/ or Hamcrest?
That would allow sentences like assertThat(players[0]).isInstanceOf(CommandLinePlayer)
and it will give muuuuch better errors than just `false`.

You can also extract stuff here to make more legible.
The ratio of 'Java Syntax' to 'meaningful words of the application' is to high for my taste...